terraform/ec2/mac/main.tf

```diff
@@ -57,15 +74,26 @@ resource "null_resource" "integration_test" {
     host        = aws_instance.cwagent.public_ip
     timeout     = "10m"
   }
+
+  provisioner "file" {
```

Comment:
Uploading the required file (in this case, amazon-cloudwatch-agent.json)  for [the agent to run later on](https://github.com/aws/amazon-cloudwatch-agent-test/pull/162/files#diff-c39bd32a8080208f0b24f4767e43af58d4eca905eb26cfcef778afec1dde7b15R114)